content/en/blog/_posts/2024-12-11-Kubernetes-v1-32-Release/index.md

```diff
@@ -211,6 +245,42 @@ workloads.
 
 This work was done as part of [KEP #4818](https://github.com/kubernetes/enhancements/issues/4818) by [SIG
 Node](https://github.com/kubernetes/community/tree/master/sig-node).
+
+### DRA: Standardized network interface data for resource claim status
+
+This enhancement adds a new field that allows drivers to report specific device status data for each allocated object 
+in a resource claim. It also establishes a standardized way to represent networking devices information.
+
+This work was done as a part of 
+[KEP #4817](https://github.com/kubernetes/enhancements/issues/4817), by 
+[SIG Network](https://github.com/kubernetes/community/tree/master/sig-network).
+
+### New statusz and flagz endpoints for core components
+
+New `/statusz` and `/flagz` endpoints added for core components (consumable via API only) to enhance debuggability 
+by gaining insight into what versions (e.g. Golang version) that component is running as, along with details 
+about its uptime, and which command line flags that component was executed with; 
+making it easier to diagnose both runtime and configuration issues.
+
+This work was done as part of 
+[KEP #4827](https://github.com/kubernetes/enhancements/issues/4827) 
+and [KEP #4828](https://github.com/kubernetes/enhancements/issues/4828) by 
+[SIG Instrumentation](https://github.com/kubernetes/community/tree/master/sig-instrumentation).
+
+### Windows strikes back!
+
+Support for graceful shutdowns of Windows nodes in Kubernetes clusters has been added. 
+Before this release, Kubernetes provided graceful node shutdown functionality for Linux nodes 
+but lacked equivalent support for Windows. This enhancement enables the kubelet on Windows nodes to handle system 
+shutdown events properly. Doing so, it ensures that Pods running on Windows nodes are gracefully terminated, 
+allowing workloads to be rescheduled without disruption. This improvement enhances the reliability and stability 
+of clusters that include Windows nodes, especially during a planned maintenance or any system updates.
+
+Moreover CPU and memory affinity support has been added for Windows nodes with CPUManager, MemoryManager and TopologyManager.
```

Comment:
Following docs style, we write those names in sentence case (otherwise they might look like API kinds)
```suggestion
Moreover CPU and memory affinity support has been added for Windows nodes, with improvements
to the CPU manager, memory manager and topology manager.
```

Reply:
Solved!